experiments/resources/back_up_optimizely.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -exuo pipefail
+
+# Required by click http://click.pocoo.org/5/python3/
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+export GIT_AUTHOR_NAME = $GIT_AUTHOR_NAME
```

Comment:
```suggestion
export GIT_AUTHOR_NAME
```
I think this should work since GIT_AUTHOR_NAME is already a variable with a value. This says export this existing variable.